Lib/importlib/_bootstrap_external.py

```diff
@@ -1033,6 +1039,21 @@ def set_data(self, path, data, *, _mode=0o666):
                                         exc)
 
 
+def _exclude_directory_from_backups(directory):
+    """Exclude the directory from backups using a CACHEDIR.TAG file.
+
+    If the file exists or there's a permission error this is a no-op.
+    """
+    try:
+        with _io.FileIO(_path_join(directory, 'CACHEDIR.TAG'), 'x') as f:
+            f.write(b"""Signature: 8a477f597d28d172789f06886806bc55
+# This file is a cache directory tag automtically created by CPython.
```

Comment:
```suggestion
# This file is a cache directory tag automatically created by CPython.
```

I am not sure if this should say Python or CPython; do other implementations create pyc directories?

Reply:
PyPy3 creates ``__pycache__`` and ``.pyc`` files, not sure about others. I'm in no way attached to CPython in this message, could be Python just as well.